docs/pages/docs/api-reference/ponder-cli.mdx

```diff
@@ -106,6 +106,19 @@ List all deployments
 │ demo          │       no │      1 day ago │           5 │
 ```
 
+## db prune
+
+Prune all inactive deployments.
```

Comment:
We could probably be more specific here and in the help text, maybe

Drop all database schemas created by Ponder instances that are no longer active.

And a similar description for the `db list` command.